testpmd-operator/roles/testpmd/templates/deployment.yml

```diff
@@ -15,6 +15,7 @@ spec:
       annotations:
 {% if high_perf_runtime is defined and high_perf_runtime|length %}
```

Comment:
As commented in https://github.com/dci-labs/example-cnf-config/pull/87, high_perf_runtime needs to be renamed to runtimeClassName var, which is the one provided in TestPMD CR: https://github.com/rh-nfv-int/nfv-example-cnf-deploy/blob/master/roles/example-cnf-app/templates/testpmd-cr.yaml.j2#L19